renderers/css/actions.py

```diff
@@ -43,6 +55,23 @@ def render_exit(self, actions_node, action_model):
         action_model._action_type = 'EXECUTE_JAVASCRIPT'
         action_model.embedded = True
         # Render the javascript
-        action_node = self.render_simple(actions_node, action_model)
+        action_node = super(OpiExit, self).render(actions_node, action_model)
         n = et.SubElement(action_node, 'scriptText')
         n.text = et.CDATA(EXIT_SCRIPT)
+
+
+class OpiActions(object):
+    """Renderer for actions."""
+
+    ACTION_MAPPING = {actions.ExecuteCommand: OpiExecuteCommand,
+                      actions.WritePv: OpiWritePv,
+                      actions.Exit: OpiExit,
+                      actions.OpenOpi: OpiOpen}
+
+    def render(self, widget_node, tag_name, action_list):
+        actions_node = et.SubElement(widget_node, tag_name)
+        for action_model in action_list:
+            t = text.OpiText()
+            c = OpiActions.ACTION_MAPPING[type(action_model)]
+            renderer = c(t)
```

Comment:
On that note, Landscape is picking up a lot of false positives. `t` and `c` are poor (I forgot to go back and clean up), but `f` for a file and `x` and `y` are not, imo.